lucien-ios/ComicDetailScreen/ComicDetailViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ComicDetailViewModel.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/24/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+import RxCocoa
+import RxSwift
+
+final class ComicDetailViewModel {
+
+    private var dashboardComicUser: DashboardComicUser
+
+    let comicImage = Variable(UIImage())
+    let comicTitle = Variable("-")
+    let storyTitle = Variable("-")
+    let volume = Variable("-")
+    let issue = Variable("-")
+    let releaseDate = Variable("-")
+    let publisher = Variable("-")
+    let genre = Variable("-")
+    let condition = Variable("-")
+
+    init(comic: DashboardComicUser) {
+        dashboardComicUser = comic
+        setObservables()
+    }
+
+    func setObservables() {
+        comicTitle.value = dashboardComicUser.dashboardComic.comicTitle
+        storyTitle.value = dashboardComicUser.dashboardComic.storyTitle
+        volume.value = dashboardComicUser.dashboardComic.volume ?? ""
```

Comment:
Should you default to an empty string or the placeholder string if the value is nil?